[2024-11-17 20:44:25,816] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 20:44:25] "POST /progressao HTTP/1.1" 200 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 20:48:11,608] INFO _log() L97   [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.1.6:5000 - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 20:48:11,610] INFO _log() L97   [33mPress CTRL+C to quit[0m - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 20:48:23,557] INFO _log() L97   [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.1.6:5000 - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 20:48:23,559] INFO _log() L97   [33mPress CTRL+C to quit[0m - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 20:48:26,716] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 20:48:26] "[32mGET / HTTP/1.1[0m" 302 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 20:48:26,777] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 20:48:26] "GET /openapi/ HTTP/1.1" 200 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 20:48:27,031] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 20:48:27] "[36mGET /openapi/rapidoc/images/rapidoc.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 20:48:27,036] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 20:48:27] "[36mGET /openapi/redoc/images/redoc.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 20:48:27,101] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 20:48:27] "[36mGET /openapi/swagger/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 20:48:27,971] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 20:48:27] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 20:48:28,004] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 20:48:28] "[36mGET /openapi/swagger/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 20:48:28,051] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 20:48:28] "[36mGET /openapi/swagger/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 20:48:28,067] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 20:48:28] "[36mGET /openapi/swagger/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 20:48:28,496] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 20:48:28] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 20:48:39,463] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 20:48:39] "POST /progressao HTTP/1.1" 200 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 20:48:41,297] WARNING add_progressao() L56   Erro ao adicionar progressão 'Texto da progressão', Progressão já existente na base :/ - call_trace=C:\Users\rafga\git\mvp-api\app.py L56  
[2024-11-17 20:48:41,298] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 20:48:41] "[31m[1mPOST /progressao HTTP/1.1[0m" 409 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 21:04:30,733] INFO _log() L97   [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.1.6:5000 - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 21:04:30,735] INFO _log() L97   [33mPress CTRL+C to quit[0m - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 21:04:37,070] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 21:04:37] "[32mGET / HTTP/1.1[0m" 302 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 21:04:37,117] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 21:04:37] "GET /openapi/ HTTP/1.1" 200 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 21:04:37,824] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 21:04:37] "[36mGET /openapi/redoc/images/redoc.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 21:04:37,826] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 21:04:37] "[36mGET /openapi/rapidoc/images/rapidoc.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 21:04:37,921] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 21:04:37] "[36mGET /openapi/swagger/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 21:04:39,815] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 21:04:39] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 21:04:39,898] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 21:04:39] "[36mGET /openapi/swagger/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 21:04:39,915] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 21:04:39] "[36mGET /openapi/swagger/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 21:04:40,009] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 21:04:40] "[36mGET /openapi/swagger/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 21:04:40,653] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 21:04:40] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 21:04:50,302] WARNING add_progressao() L56   Erro ao adicionar progressão 'Texto da progressão', Progressão já existente na base :/ - call_trace=C:\Users\rafga\git\mvp-api\app.py L56  
[2024-11-17 21:04:50,303] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 21:04:50] "[31m[1mPOST /progressao HTTP/1.1[0m" 409 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 21:04:51,702] WARNING add_progressao() L56   Erro ao adicionar progressão 'Texto da progressão', Progressão já existente na base :/ - call_trace=C:\Users\rafga\git\mvp-api\app.py L56  
[2024-11-17 21:04:51,705] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 21:04:51] "[31m[1mPOST /progressao HTTP/1.1[0m" 409 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 21:05:00,001] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 21:05:00] "POST /progressao HTTP/1.1" 200 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 21:05:09,540] INFO _log() L97   127.0.0.1 - - [17/Nov/2024 21:05:09] "POST /progressao HTTP/1.1" 200 - - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-11-17 21:05:18,638] ERROR log_exception() L875  Exception on /progressoes [GET] - call_trace=C:\Users\rafga\git\mvp-api\env\Lib\site-packages\flask\app.py L875 
Traceback (most recent call last):
  File "C:\Users\rafga\git\mvp-api\env\Lib\site-packages\flask\app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\rafga\git\mvp-api\env\Lib\site-packages\flask\app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\rafga\git\mvp-api\env\Lib\site-packages\flask_cors\extension.py", line 194, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ~^^^^^^^^^^^^^^^^^
  File "C:\Users\rafga\git\mvp-api\env\Lib\site-packages\flask\app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\rafga\git\mvp-api\env\Lib\site-packages\flask\app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\rafga\git\mvp-api\env\Lib\site-packages\flask_openapi3\scaffold.py", line 117, in view_func
    response = func(**func_kwargs)
  File "C:\Users\rafga\git\mvp-api\app.py", line 77, in get_progressoes
    if not progressoes:
           ^^^^^^^^^^^
NameError: name 'progressoes' is not defined. Did you mean: 'progressao'?
